images/krte/cloudbuild.yaml

```diff
@@ -28,6 +28,8 @@ substitutions:
   _CFSSL_VERSION: R1.2
 options:
   substitution_option: ALLOW_LOOSE
+  # this is a large and critical CI image, builds are slow on the default 1 core
+  machineType: N1_HIGHCPU_32
```

Comment:
this takes longer to startup (there's a pre warmed pool of default machines) but is *much* more powerful

Reply:
https://cloud.google.com/build/docs/build-config-file-schema#options

Reply:
https://github.com/kubernetes/ingress-nginx/issues/9813 and switched to e2 here